crowbar_framework/app/models/node_object.rb

```diff
@@ -1451,6 +1451,31 @@ def disk_release!(device, owner)
     disk_release(device, owner) and save
   end
 
+  def verify_claimed_disks
```

Comment:
I think it depends on the caller and how it handles the returned Array of disks, then maybe `verified_claimed_disks`? I am not sure what's best here but if you think it's ok like it is now I trust you there.